scipy/stats/stats.py

```diff
@@ -2395,15 +2453,21 @@ def zmap(scores, compare, axis=0, ddof=0):
     >>> b = [0, 1, 2, 3, 4]
     >>> zmap(a, b)
     array([-1.06066017,  0.        ,  0.35355339,  0.70710678])
+    
     """
     scores, compare = map(np.asanyarray, [scores, compare])
-    mns = compare.mean(axis=axis, keepdims=True)
-    sstd = compare.std(axis=axis, ddof=ddof, keepdims=True)
-    return (scores - mns) / sstd
+    mns = compare.mean(axis=axis)
```

Comment:
Did you change the code or is it from a merge? I think it is better to split code changes and doc changes